visualization/Readme.md

```diff
@@ -0,0 +1,110 @@
+This is a tool for data analysis, consisting of two pages: "Inference" and "Analyze".
+
+### Getting Started
+Before using this tool, follow the instructions in [prerequisites.md](/docs/prerequisites.md), download datasets, and install requirements:
+```shell
+pip install -r visualization/requirements.txt
+./datasets/prepare_all.sh
+```
+You can adjust parameters in the [visualization_config.yaml](/visualization/settings/visualization_config.yaml)  file or via the command line. Use the following command to launch the program (all parameters are optional):
+```shell
+python visualization/data_explorer.py \
```

Comment:
I think the main use-case is actually running this locally, not on slurm - so maybe list just the host as an argument and mention that can also add ssh parameters if this is necessary for connecting to slurm